base-notebook/binder/environment.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - dask-kubernetes>=0.8.0
   - nbserverproxy
   - ipywidgets
+  - graphviz
```

Comment:
Happy to try this again but I have a seem to remember this not working in a recent test. We'll see.